pages/storage.vue

```diff
@@ -120,7 +120,9 @@ export default {
       q.answers.find((answer) => answer.answer === q.default_answer)
     );
 
-    return { tool, answers };
+    return { tool, answers }.catch((e) =>
+      error({ statusCode: 404, message: 'Page not found' })
+    );
```

Comment:
I don't think this one should/can error out, but if it can, the catch should be after the fetch on line 112